test/e2e/specs/site-editor/site-editor-inserter.spec.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+const { test } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Block list view', () => {
```

Comment:
Copy-paste error?

```suggestion
test.describe( 'Site Editor Inserter', () => {
```

Reply:
Yes, it was a copy-paste mistake I was nervous. thanks